flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/metrics/MySqlSourceReaderMetrics.java

```diff
@@ -18,17 +18,42 @@
 package org.apache.flink.cdc.connectors.mysql.source.metrics;
 
 import org.apache.flink.cdc.connectors.mysql.source.reader.MySqlSourceReader;
+import org.apache.flink.metrics.Counter;
 import org.apache.flink.metrics.Gauge;
 import org.apache.flink.metrics.MetricGroup;
 import org.apache.flink.runtime.metrics.MetricNames;
 
+import io.debezium.relational.TableId;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
 /** A collection class for handling metrics in {@link MySqlSourceReader}. */
 public class MySqlSourceReaderMetrics {
 
     public static final long UNDEFINED = -1;
 
     private final MetricGroup metricGroup;
 
+    public static final String IO_NUM_RECORDS_OUT_SNAPSHOT = ".numRecordsOutBySnapshot";
+
+    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_INSERT =
+            ".numRecordsOutByDataChangeEventInsert";
+
+    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_DELETE =
+            ".numRecordsOutByDataChangeEventDelete";
+
+    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_UPDATE =
+            ".numRecordsOutByDataChangeEventUpdate";
```

Comment:
I suggest to remove dot(.) from metric name.

```suggestion
    public static final String IO_NUM_RECORDS_OUT_SNAPSHOT = "numRecordsOutBySnapshot";

    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_INSERT =
            "numRecordsOutByDataChangeEventInsert";

    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_DELETE =
            "numRecordsOutByDataChangeEventDelete";

    public static final String IO_NUM_RECORDS_OUT_DATA_CHANGE_EVENT_UPDATE =
            "numRecordsOutByDataChangeEventUpdate";
```

Reply:
Ok, thank you for helping me review the code, I will remove (.)